### PR TITLE
Add a "Getting Started" section.

### PR DESCRIPTION
**Pull request by slightlyoff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,60 @@ render(helloTemplate('Kevin'), document.body);
...
+```
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <script
+        type="module"
+        src="node_modules/lit-html/lit-html.js"></script>
+  </head>
```
Comment (arthurevans) [inline README.md]: Does this create globals you can use to access `html` and `render`? I've only used the `import {html, render} ...` form.

File: README.md [hunk 2]
```diff
@@ -37,6 +37,60 @@ render(helloTemplate('Kevin'), document.body);
...
+
+If you'd prefer not to use NPM, hot-loading Lit from [`unpkg.com`] is straightforward:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <script
+        type="module"
+        src="https://unpkg.com/lit-html@0.7.1/lib/lit-extended.js?module"
```
Comment (jridgewell) [inline README.md]: `//unpkg.com`

File: README.md (hunk 1 above)
Comment (jridgewell) [reply README.md]: Yah, where do exported methods go?

File: README.md (hunk 2 above)
Comment (JakeChampion) [reply README.md]: I think preferring https always is what is recommended now.

**Comment by justinfagnani** [comment]
Going to close this as part of the current documentation migration to polymer.github.io/lit-html/. There's a getting started section there.

**Pull request closed without merging**